Deartoday/Deartoday/Screen/Splash/Controller/SplashViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SplashViewController.swift
+//  Deartoday
+//
+//  Created by 황찬미 on 2022/07/18.
+//
+
+import UIKit
+
+import Lottie
+import SnapKit
+import Then
+
+final class SplashViewController: UIViewController {
+    
+    // MARK: - UI Property
+    
+    var soundOnImageView = UIImageView().then {
+        $0.image = Constant.Image.soundon
+        $0.alpha = 0
+    }
+    
+    // MARK: - Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        showSplash()
+        setLayout()
+        setAnimation()
+    }
+    
+    // MARK: - Custom Method
+    
+    private func showSplash() {
+        let spashScreen = AnimationView(name: Constant.Lottie.splash)
+        spashScreen.frame = self.view.bounds
+        spashScreen.center = self.view.center
+        spashScreen.contentMode = .scaleAspectFill
+        self.view.addSubview(spashScreen)
+        spashScreen.play()
+    }
+    
+    private func setLayout() {
+        view.addSubview(soundOnImageView)
+        
+        soundOnImageView.snp.makeConstraints {
+            $0.leading.trailing.equalToSuperview().inset(6)
+            (getDeviceHeight() == 667) ? $0.bottom.equalToSuperview().inset(70) : $0.bottom.equalToSuperview().inset(90)
```

Comment:
좋은데 혹시

```suggestion
            $0.bottom.equalToSuperview().inset( getDeviceHeight() == 667 ? 70 : 90)
```

이거 대나요? 안될라나?
저거도 조아용 ~!! 굿!

Reply:
헤헷.
경민이처럼 말랑한 사고를 가집시다.